src/fixate/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.5.7"
+__version__ = "0.5.8"
```

Comment:
master has moved on again. Can you merge master again and resolve any remaining conflicts